src/components/Icons/PlusIcon.tsx

```diff
@@ -0,0 +1,5 @@
+import { ReactComponent as PlusSvg } from '../../assets/Icons/Plus.svg';
+
+export const PlusIcon: React.FC = () => {
+  return <PlusSvg />;
+};
```

Comment:
What's the purpose of creating a component from another component?

Reply:
none really.
but I think we need a wrapper for if we want to use themIcon of mantine, right?

Reply:
Let's not do it right now.